From 4367334cfc37ce457ac703df1e8073f6d262765b Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Fri, 12 Jul 2013 15:44:18 +0800
Subject: [PATCH 2636/2666] ENGR00269468 IPUv3 dev:Silence an annoying warning
 message

We call the function ipu_calc_stripes_sizes() to calculate
stripe settings for vdi split mode. Chances are that the
parameters output_frame_width and maximal_stripe_width for the
function satisfy the relationship 'output_frame_width <=
maximal_stripe_width' and make the function return non-zero
value. This causes the IPU device driver generates an annoying
warning message, though the default stripe settings still can
work. This patch simply silences the warning message by reducing
the print log level from error to debug.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 58d6cb0f2d5013c7a6f7b163ce8834019f0dcbc0)
---
 drivers/mxc/ipu3/ipu_device.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mxc/ipu3/ipu_device.c b/drivers/mxc/ipu3/ipu_device.c
index b10ce2a..ce34775 100644
--- a/drivers/mxc/ipu3/ipu_device.c
+++ b/drivers/mxc/ipu3/ipu_device.c
@@ -852,7 +852,7 @@ static int update_split_setting(struct ipu_task_entry *t, bool vdi_split)
 				&left_stripe,
 				&right_stripe);
 		if (ret)
-			dev_err(t->dev, "Warn: no:0x%x,calc_stripes ret:%d\n",
+			dev_dbg(t->dev, "Warn: no:0x%x,calc_stripes ret:%d\n",
 				 t->task_no, ret);
 		t->set.sp_setting.iw = left_stripe.input_width;
 		t->set.sp_setting.ow = left_stripe.output_width;
-- 
1.7.7.4

